packagedb/api.py

```diff
@@ -818,6 +818,10 @@ def list(self, request, format=None):
         purl = validated_data.get('purl')
 
         kwargs = dict()
+        # We want this request to have high priority since the user knows the
+        # exact package they want
+        kwargs['priority'] = 100
```

Comment:
Just curious, so now there's only two values for priority 100 for packages with versions and 0 otherwise? Are there other cases where we set priority?

Reply:
This is the only place where we set priority for things on the scan queue. Before, I think everything had a priority of 0.